src/main/java/mat/client/shared/MeasureLibraryGridToolbar.java

```diff
@@ -0,0 +1,163 @@
+package mat.client.shared;
+
+import java.util.Collection;
+
+import org.gwtbootstrap3.client.ui.Button;
+import org.gwtbootstrap3.client.ui.constants.ButtonSize;
+import org.gwtbootstrap3.client.ui.constants.ButtonType;
+import org.gwtbootstrap3.client.ui.constants.IconType;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.thirdparty.guava.common.annotations.VisibleForTesting;
+import mat.client.measure.ManageMeasureSearchModel;
+
+public class MeasureLibraryGridToolbar extends HorizontalFlowPanel {
+
+    private Button versionButton;
+    private Button historyButton;
+    private Button editButton;
+    private Button shareButton;
+    private Button cloneButton;
+    private Button exportButton;
+
+    public MeasureLibraryGridToolbar() {
+        setStyleName("action-button-bar");
+        addStyleName("btn-group");
+        addStyleName("btn-group-sm");
+
+        versionButton = GWT.create(Button.class);
+        historyButton = GWT.create(Button.class);
+        editButton = GWT.create(Button.class);
+        shareButton = GWT.create(Button.class);
+        cloneButton = GWT.create(Button.class);
+        exportButton = GWT.create(Button.class);
+
+        add(versionButton);
+        add(historyButton);
+        add(editButton);
+        add(shareButton);
+        add(cloneButton);
+        add(exportButton);
```

Comment:
Not sure if the order of the buttons is fine. Wireframe shows different order

Reply:
I changed it to resemble the CQL toolbar. I will clarify it with Kathy